### PR TITLE
Refactor style properties of the 'Code instrumentation' subsystem

### PR DESCRIPTION
**Pull request by LavrovArtem**
part of #1379

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: src/client/sandbox/node/element/index.js [hunk 1]
```diff
@@ -36,6 +37,7 @@ export default class ElementSandbox extends SandboxBase {
         this.uploadSandbox = uploadSandbox;
         this.iframeSandbox = iframeSandbox;
         this.eventSandbox  = eventSandbox;
+        this.styleSandbox  = new StyleSandbox(this);
```
Comment (miherlosev) [inline src/client/sandbox/node/element/index.js]: Another sandboxes instantiated in `sandbox.js` and  attached in `sandbox.attach` method.
Do same for the `StyleSandbox`.

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: src/client/sandbox/node/element/style.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    constructor () {
+        super();
+
+        this.URL_PROPS        = ['background', 'backgroundImage', 'borderImage',
+            'borderImageSource', 'listStyle', 'listStyleImage', 'cursor'];
+        this.DASHED_URL_PROPS = StyleSandbox._generateDashedProps(this.URL_PROPS);
+
+        // NOTE: The CSS2Properties class is supported only in the Firefox
+        // and its prototype contains all property descriptors
+        this.isProtoContainsAllProps = !!window.CSS2Properties;
```
Comment (miherlosev) [inline src/client/sandbox/node/element/style.js]: 1)Declare boolean props without `is` suffix.

2)These flags are feature detection related only to styles. I propose to combine them to the single variable.
```js
constructor () {
   ....
   StyleSandbox._detectBrowserFeatures (); // or just '._detectFeatures'
 
and inside this function fill the variable:

this.features = {
    protoContainsAllProps:...,
    protoContainsUrlDescriptors: ....
    ....
};
  ...
}
```

File: src/client/sandbox/node/element/index.js (hunk 1 above)
Comment (LavrovArtem) [reply src/client/sandbox/node/element/index.js]: NodeSandbox
https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/node/index.js#L34-L36

**Comment by miherlosev** [comment]
Let's wait for green test and run health monitor for this PR.

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Pull request merged**